proteus/Domain.py

```diff
@@ -1,7 +1,55 @@
 """
 A class hierarchy and tools for building domains of PDE's.
 """
+import numpy as np
+import BC
+from MeshTools import MeshParallelPartitioningTypes as mpt
 
+class Mesh:
```

Comment:
There is already a Mesh class in MeshTools. I would name this class MeshOptions.